hearth/lib/hearth/middleware/send.rb

```diff
@@ -50,9 +54,17 @@ def apply_stub(stub, context, output)
         when Class
           output.error = stub.new
         when Hash
-          @stub_class.stub(context.response, stub: stub)
+          @stub_class.stub(context.response,
+                           stub: @stub_params_class.build(
+                             stub,
+                             context: "stub[#{context.operation_name}]"
+                           ))
```

Comment:
These aren't very readable... maybe split it out on multiple lines?

Reply:
I had them more readable but rubocop complained and autocorrected to this.  But updated to something else.